src/entry-points/popup/App.svelte

```diff
@@ -438,14 +468,68 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
       marginAfter: 0.03 + 0.0020 * (100 - settings.simpleSlider)
     })
   }
+
+  let hotkeysActions: {[P in HotkeyAction]?: HotkeyBinding[]} = {};
+  settingsPromise.then((settings) => {
+    if (!settings.enableHotkeys) return;
+    [...settings.hotkeys, ...settings.popupSpecificHotkeys].forEach((hotKey) => {
+      const actionId = hotKey.action;
+      if (!hotkeysActions[actionId]) hotkeysActions[actionId] = [];
+      hotkeysActions[actionId].push(hotKey);
+    });
+  });
+
+  function getActionString(actionId: HotkeyAction, actionName: string): string {
+    const actionHotkeys = hotkeysActions[actionId];
+    if (!actionHotkeys) return '';
+    let actionString = '';
+    actionHotkeys.forEach((hotkey: HotkeyBinding, i: number) => {
+      let keysString = '';
+      const modifiers = hotkey.keyCombination.modifiers;
+      if (modifiers) {
+        modifiers.forEach((modifier) => {
+          keysString += modifier.replace('Key', '') + '+';
+        });
+      }
+              
+      actionString += keysString + hotkey.keyCombination.code.replace('Key', '');
+      if (actionHotkeys.length !== i + 1) {
+        actionString += ', ';
+      }
+    });
+
+    return '\n' + actionName + ': ' + actionString;
```

Comment:
I think the new strings need to utilize interpolation, e.g. as in `applyToOnly`. But this is probably fine for most languages.